### PR TITLE
RD-1123 Changed Type column to boolean System column

### PR DESCRIPTION
**Pull request by kubama**
image

I'm not planning system tests execution - I run the updated test locally and it passes.

### CODE REVIEW

File: widgets/filters/src/FiltersTable.tsx [hunk 1]
```diff
@@ -63,15 +63,15 @@ const FiltersTable: FunctionComponent<FiltersTableProps> = ({ data, toolbox, wid
...
                 <DataTable.Column width="112px" />
                 {data.filters.map(filter => (
                     <DataTable.Row key={filter.id}>
                         <DataTable.Data>{filter.id}</DataTable.Data>
                         <DataTable.Data>{filter.created_by}</DataTable.Data>
                         <DataTable.Data>{Time.formatTimestamp(filter.created_at)}</DataTable.Data>
-                        <DataTable.Data>
-                            {i18n.t(`widgets.filters.type.${filter.is_system_filter ? 'system' : 'user'}`)}
+                        <DataTable.Data className="center aligned">
+                            <Checkbox checked={filter.is_system_filter} disabled />
```
Comment (qooban) [inline widgets/filters/src/FiltersTable.tsx:74]: Optional/General 🙂 

Looking at the search:
image
that could probably be extracted to a separate component (sth like `CenteredData` or `CenterAlignedData`), but:
1. I'm not sure `DataTable` would be able to pick it up and handle it properly (not sure if it does not require to have `DataTable.Data` just under `Data.TableRow` component)
2. It's out of the scope of this PR

**Review by qooban — APPROVED** [review]
All good 👍🏻

File: widgets/filters/src/FiltersTable.tsx (hunk 1 above)
Comment (kubama) [reply widgets/filters/src/FiltersTable.tsx:74]: Personally I would rather try to add a new prop to `DataTable.Data` that would set these classes.

Comment (qooban) [reply widgets/filters/src/FiltersTable.tsx:74]: Yep, make sense. Created RD-2083.

**Pull request merged**